lib/apiservers/engine/backends/archive.go

```diff
@@ -249,9 +229,15 @@ func NewArchiveStreamWriterMap(mounts []types.MountPoint, containerDestPath stri
 		// file data.txt from local /mnt/A/data.txt will come to the persona as mnt/A/data.txt.
 		// Here, we must tell the portlayer to remove "mnt/A".  The key to determining whether to
 		// strip "A" or "mnt/A" is based on the container destination path.
-		if containerDestPath != "/" && strings.HasPrefix(aw.mountPoint.Destination, containerDestPath) {
-			aw.filterSpec.StripPath = strings.TrimPrefix(aw.mountPoint.Destination, containerDestPath)
-		}
+
+		// hickeng: the current logic is strip, then rebase for export - more efficient to collapse
+		// any reducdency here than apply to every tar entry when packing but skipping for now.
+		aw.filterSpec.StripPath = aw.mountPoint.Destination
+		aw.filterSpec.RebasePath = strings.TrimPrefix(dest, aw.mountPoint.Destination)
```

Comment:
This should get fixed with the spec generation PR that I have out(https://github.com/vmware/vic/pull/5720). Currently this will generate invalid spec for things like sub volume targets and sub volume CopyTo/CopyFroms that would target the root of the volume itself.